### PR TITLE
[#7133] Add test for ichksum -n (main)

### PR DESCRIPTION
**Pull request by alanking**
Addresses #7133

### CODE REVIEW

File: scripts/irods/test/test_targeting_specific_replica_number.py [hunk 1]
```diff
@@ -354,3 +354,44 @@ def test_istream_n_always_overwrites_and_reads_requested_replica(self):
...
+        self.admin.assert_icommand(['iadmin', 'modresc', self.pt2, 'context', 'read=0.1'])
+
+        self.user.assert_icommand(['ils', '-L', self.logical_path], 'STDOUT') # debugging
+
+        # execute ichksum -n 1, assert that replica 1 has a checksum even though it will vote extremely low.
+        self.user.assert_icommand(['ichksum', '-n1', self.logical_path], 'STDOUT', '    sha2:')
+
+        # Assert that replica 0 has no checksum and replica 1 has a checksum.
+        self.assertEqual(len(lib.get_replica_checksum(self.user, data_name, 0)), 0)
+        self.assertNotEqual(len(lib.get_replica_checksum(self.user, data_name, 1)), 0)
```
Comment (korydraughn) [inline scripts/irods/test/test_targeting_specific_replica_number.py:397]: Feel free to ignore this, but would it be better to assert then length is greater than zero?

**Review by korydraughn — APPROVED** [review]
Tests look good.

If they pass at the bench, then add the pound and merge.

**Review by korydraughn — APPROVED** [review]

**Comment by korydraughn** [comment]
Please squash and pound if the tests pass.

**Comment by alanking** [comment]
They do pass, indeed.

**Pull request merged**